### DISCUSSION TITLE
CVODE support

### DISCUSSION THREAD

**Discussion by bnkeeton** [Nek5000/nekRS#559] [General]
Hi, I'm interested in using CVODE with nekrs. After compiling nekrs with `-DENABLE_CVODE=ON`, I'm trying to run the lowMach example with `ciMode = 3`. The case completes, and in the logfile I see `key: SCALAR00 SOLVER` is set to CVODE, but it appears there is no solution to this scalar (temperature in this case). I confirmed this by running a separate case (simple 2D jet) with the same `options.setArgs`.

Is there anything else needed to get a CVODE case working, or is there a working example with CVODE activated?

Thanks,
Ben Keeton

**Comment by bnkeeton** [comment]
I'm just following up on this post. I checked the `next` branch and it appears cvode is not yet implemented. I tried `-DENABLE_CVODE=ON` on Polaris but don't see any plugins loaded when I build nekRS:

![image](https://github.com/user-attachments/assets/af1ec6ba-4dcc-47c3-9aa1-8a1fc8364d99)

Is that correct? If so, is there an estimate on when it might be available in nekRS?

Thanks,
Ben

**Reply by stgeke** [reply]
ETA is expected to be sometime next year.

**Reply by bnkeeton** [reply]
Great, thanks for the feedback.

Ben
